pkg/cli/serve/serve.go

```diff
@@ -94,14 +93,17 @@ func NewServeCmd() *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringVarP(&o.ocmURL, ocmURLFlagName, "", "", "OCM URL")
-	cmd.Flags().StringVarP(&o.services, servicesFlagName, "", "", "OCM service name")
-	cmd.Flags().StringVarP(&o.accessToken, accessTokenFlagName, "t", "", "Access token for OCM")
-	cmd.PersistentFlags().BoolVarP(&o.debug, debugFlagName, "d", false, "Debug mode enable")
+	cmd.Flags().StringVarP(&o.ocmURL, config.OcmURL, "", "", "OCM URL")
+	cmd.Flags().StringVarP(&o.services, config.Services, "", "", "OCM service name")
+	cmd.Flags().StringVarP(&o.accessToken, config.AccessToken, "t", "", "Access token for OCM")
+	cmd.Flags().StringVarP(&o.clusterID, config.ClusterID, "i", "", "Cluster ID")
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&o.clusterID, config.ClusterID, "c", "", "Cluster ID")
```
Minor point.. Am thinking if 'c' for cluster is more intuitive than 'i' for cluster ID?

Reply:
That makes sense and would be more consistent with `ocm` and `rosa` CLIs - thanks I have made that change now.